tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -77,6 +90,87 @@ module.exports = {
       }
       return this.api.page.FilesPageElement.filesList()
     },
+    changeCustomPermissionsTo: async function (user, requiredPermissions) {
+      requiredPermissions = requiredPermissions.replace(/\s/g, '')
+      const requiredPermissionArray = requiredPermissions.split(',')
+      const informationSelector = util.format(this.elements.collaboratorInformationByCollaboratorName.selector, user)
+      const everyPermissions = ['share', 'change', 'create', 'delete']
+
+      await this
+        .useXpath()
+        .waitForElementPresent(informationSelector)
+        .click(informationSelector)
+        .waitForElementVisible('@selectRoleButtonInCollaboratorInformation')
+        .click('@selectRoleButtonInCollaboratorInformation')
+        .waitForElementVisible('@collaboratorCustomRole')
```

Comment:
no need to select the custom-role here, change the permissions on what-ever role is currently selected